code/game/objects/items/weapons/weaponry.dm

```diff
@@ -530,10 +534,16 @@ obj/item/weapon/banhammer/admin
 	icon_state = "spikebat"
 	item_state = "spikebat"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/swords_axes.dmi', "right_hand" = 'icons/mob/in-hand/right/swords_axes.dmi')
+	flags = TWOHANDABLE
 	force = 15
 	sharpness = 0.5
 	sharpness_flags = SHARP_TIP
 	throwforce = 10
 	throw_speed = 1
 	throw_range = 7
 	w_class = W_CLASS_LARGE
+
+/obj/item/weapon/spiked_bat/update_wield()
+	..()
+	force = wielded ? 25 : initial(force)
```

Comment:
25?

Reply:
I think I'm going to reduce the spiked bat's base force to 12 and wielded to 15 once I get the in-hand wielded sprites. You were right that the sharpness makes it a bit too powerful.